### DISCUSSION TITLE
Who is using freetube all the time, like me?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [FreeTubeApp/FreeTube#3058] [General]
I installed freetube on my mini PC to watch on my TV.
I thought it would never replace the youtube site. 

Well, I always use freetube for months. I have no more youtube account (been banned and finally never bothered to make another)

I'm missing some features deeply (like playlists) but I love it.
Just wanted to make a shoutout

And who is in my case?

**Comment by ssnor** [comment]
I'm with you, cant remember when I used the actual youtube.com site last.
Must have been years at this point. Subscriptions without an account is a "killer app" for me.
